hitl/run.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/bash -e
```

Comment:
```suggestion
#!/bin/sh
```

For compatibility and fewer assumptions.